elasticsearch-agent/src/main/java/com/epam/pipeline/elasticsearchagent/service/impl/EntitySynchronizer.java

```diff
@@ -38,38 +38,39 @@
     private final EventToRequestConverter converter;
     private final ElasticIndexService indexService;
     private final BulkRequestSender bulkRequestSender;
+    private final int chunkSize;
 
     @Override
     @SuppressWarnings("PMD.AvoidCatchingGenericException")
     public void synchronize(final LocalDateTime lastSyncTime, final LocalDateTime syncStart) {
-        try {
-            log.debug("Starting to synchronize {} entities", objectType);
-            final List<PipelineEvent> pipelineEvents = pipelineEventDao
-                    .loadPipelineEventsByObjectType(objectType, syncStart);
-
-            log.debug("Loaded {} events for {}", pipelineEvents.size(), objectType);
-            final List<PipelineEvent> mergeEvents = EventProcessorUtils.mergeEvents(pipelineEvents);
-            if (mergeEvents.isEmpty()) {
-                log.debug("{} entities for synchronization were not found.", objectType);
-                return;
-            }
-
-            log.debug("Merged {} events for {}", mergeEvents.size(), objectType);
+        log.debug("Starting to synchronize {} entities", objectType);
+        final List<PipelineEvent> pipelineEvents =pipelineEventDao
+                    .loadPipelineEventsByObjectType(objectType, syncStart, chunkSize);
+        log.debug("Loaded {} events for {}", pipelineEvents.size(), objectType);
+        final List<PipelineEvent> mergeEvents = EventProcessorUtils.mergeEvents(pipelineEvents);
+        if (mergeEvents.isEmpty()) {
+            log.debug("{} entities for synchronization were not found.", objectType);
+            return;
+        }
+        log.debug("Merged {} events for {}", mergeEvents.size(), objectType);
 
-            final String indexName = converter.buildIndexName();
+        final String indexName;
```

Comment:
Do not split variable declaration and assigment

Reply:
Done